client/src/InfoLab/InfoLab.js

```diff
@@ -28,6 +30,13 @@ export default class InfoLab extends React.Component {
         <TM k="lab_title" el="h1" />
         <div className="medium_panel_text">
           <TM k="lab_intro_text" />
+          <button
```

Comment:
Maybe we should disable the survey button in PageDetails on InfoLab page
(I did the same for about route #766 )

Reply:
👍 I changed it back so that it still shows the report a problem and github version banners